lib/client.rb

```diff
@@ -208,6 +214,13 @@ def alive?
     false
   end
 
+  def prevent_not_ready
```

Comment:
I would rename to "reset_to_ready_state" or something similar.

Reply:
On it...